src/ioctls/system.rs

```diff
@@ -584,6 +600,14 @@ mod tests {
         }
     }
 
+    #[test]
+    #[cfg(target_arch = "aarch64")]
+    fn test_guest_debug_hw_capacity() {
+        let kvm = Kvm::new().unwrap();
+        assert!(kvm.get_guest_debug_hw_bps() >= 0);
+        assert!(kvm.get_guest_debug_hw_wps() >= 0);
```

Comment:
I think it's okay to check it as it is now in the test because we don't know what hosts we're running on. Can you add a comment about why we're doing the check this way? (i.e. different values on different platforms).